vcd/validate_funcs.go

```diff
@@ -193,3 +195,42 @@ func IsFloatAndBetween(min, max float64) schema.SchemaValidateFunc {
 func validateAlphanumericWithUnderscoresAndHyphens() schema.SchemaValidateFunc {
 	return validation.StringMatch(regexp.MustCompile(`(?i)^[a-z0-9_-]+$`), "only alphanumeric characters, underscores and hyphens allowed")
 }
+
+// IsBoolAndTrue returns a SchemaValidateFunc which tests if the provided value is convertable to
+// float64 and is set to true.
```

Comment:
Leftover? This doesn't seem to have anything related to `float64`

Reply:
Forgot to remove this code. Removed now.